src/components/DeadPlayerReviveButton.js

```diff
@@ -2,19 +2,40 @@ import React, { useState } from 'react';
 import { Image, Box} from '@chakra-ui/react';
 import revive from '../assets/revive-gray.png';
 import reviveHover from '../assets/revive-white.png';
-import PlayerRevive from './PlayerRevive';
 import CreateAlert from './CreateAlert';
+import { updateIsAliveForPlayer } from './dbCalls';
+import RemapPlayers from './RemapPlayers';
 
-const DeadPlayerReviveButton = ({player, roomID, arrayOfAlivePlayers, handlePlayerRevive, handleRemapping}) => {
+const DeadPlayerReviveButton = (props) => {
+    const {player, 
```

Comment:
The amount of props here is kind of ballooning, as you can see on your other components as well. It works fine, but it is feeling a bit overkill to be passing these handler functions into all these child components. Can we rewrite this so that instead of passing the handler functions downwards, instead we use eventing? 

Basically, wherever you do a revive or whatever, let's throw an event of the appropriate name and with the relevant details, and create a listener on GameMasterView which listens for that event and does the appropriate logic all on GameMasterView?